.github/workflows/test.yml

```diff
@@ -9,13 +9,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.x
-      uses: actions/setup-python@v2
-      with:
-        # Semantic version range syntax or exact version of a Python version
-        python-version: '3.x'
+    - name: Build Docker image
       run: |
-        python -m pip install --upgrade pip
+        make install
     - name: Test
       run: |
-        make local-test
\ No newline at end of file
+        make test
```

Comment:
Github display end of files with tis icon:
![image](https://user-images.githubusercontent.com/2212144/105716134-b7aa9700-5f1e-11eb-8953-ccb11dfecc83.png)

I don't like it too much, can you add an empty line ? If you use VSCode you can add this settings `"files.insertFinalNewline": true,`.